### PR TITLE
GROK-16449: Help: Diff Studio

### PR DESCRIPTION
**Pull request by vmakarichev**

### CODE REVIEW

File: help/compute/diff-studio.md [hunk 1]
```diff
@@ -35,11 +35,29 @@ To run **Diff Studio**:
...
+* Click **Examples** and explore built-in models. They cover all Diff Studio capabilities.
+* Find recently opened models in **Recent**.
+
+![Diff Studio Browsing](pics/diff-studio-browsing.gif)
+
+In **Browse**, click any *ivp*-file. Model preview opens. Modify inputs and check results. Use URL to share computations:
+
+![Diff Studio Sharing](pics/diff-studio-sharing-url.gif)
+
+You can apply URL to sharing runs of models from the **Templates**, **Examples** and **Recent** groups.
```
Comment (skalkin) [inline help/compute/diff-studio.md:59]: "Apply url"? Let's rephrase

File: help/compute/diff-studio.md [hunk 2]
```diff
@@ -35,11 +35,29 @@ To run **Diff Studio**:
 
 ### Loading and saving data
 
-* **To save formulas** to a local file, click the <i class="fas fa-save"></i> **Save** icon on the ribbon, and find the *ivp*-file in Downloads. You can open and edit this file using any text editor.
+* **To save formulas**, click the <i class="fas fa-save"></i> **Save** icon on the ribbon. The context menu opens:
```
Comment (skalkin) [inline help/compute/diff-studio.md:38]: Probably let's use "model" instead of "formulas" consistently

**Review by skalkin — APPROVED** [review]

File: help/compute/diff-studio.md (hunk 2 above)
Comment (vmakarichev) [reply help/compute/diff-studio.md:38]: Fixed

File: help/compute/diff-studio.md (hunk 1 above)
Comment (vmakarichev) [reply help/compute/diff-studio.md:59]: Fixed

**Pull request merged**